OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -319,7 +322,8 @@ Action InitDisplayPanel(Widget panel)
 
 			var restartDesc = panel.Get("RESTART_REQUIRED_DESC");
 			restartDesc.IsVisible = () => ds.Mode != OriginalGraphicsMode || ds.VideoDisplay != OriginalVideoDisplay || ds.GLProfile != OriginalGLProfile ||
-				(ds.Mode == WindowMode.Windowed && (origWidthText != windowWidth.Text || origHeightText != windowHeight.Text));
+				(ds.Mode == WindowMode.Windowed && (origWidthText != windowWidth.Text || origHeightText != windowHeight.Text))
+				|| ds.Language != OriginalLanguage;
```

Comment:
Style nit: We have the `||` at the end of the previous line above, I'd prefer we stick to that approach here, if only for consistency.